packet/models.py

```diff
@@ -20,7 +20,7 @@ class Freshman(db.Model):
     __tablename__ = "freshman"
     rit_username = Column(String(10), primary_key=True)
     name = Column(String(64), nullable=False)
-    onfloor = Column(Boolean, nullable=False)
+    onfloor = Column(Boolean, nullable=False)       # TODO: I don't think we need this column but I'm leaving it for now
```

Comment:
Why might we not need this? We need to know which freshmen are on floor

Reply:
Unless I'm thinking about it wrong we don't. The only thing we need to know is what the onfloor freshmen are for a given packet which is already covered by the `FreshSignature` table.